.github/workflows/aws-deploy.yaml

```diff
@@ -0,0 +1,41 @@
+name: aws-deploy
+
+on: push
+
+env:
+  AWS_ACCESS_KEY_ID: ${{ secrets.NEXTFLOW_PROD_CI_ACCESS_KEY }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.NEXTFLOW_PROD_CI_SECRET_ACCESS_KEY }}
+
+jobs:
+  pre-commit:
+    name: Run pre-commit hooks against all files
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - uses: pre-commit/action@v2.0.2
+
+  sceptre-validate:
```

Comment:
i believe `sceptre validate` just calls `aws cloudformation validate` which requires access to AWS.  That's gonna fail here since this job won't have access to AWS.  This will work if you run as post merge step though (i.e. add `if: github.ref == 'refs/heads/prod'`).  

With that being said I think you don't really need validate at all since it's basically doing the same thing as cfn-lint which is already being run with pre-commit.

Reply:
I also found out that `sceptre validate` doesn't handle hooks, so I removed this step from the CI workflow.